### ISSUE TITLE
keypoints locate on same position

### ISSUE THREAD

**Issue by acmff22** [princeton-vl/pytorch_stacked_hourglass#12]
I train the network using my own dataset to detect keypoints on objects, but during test stage, detected keypoints locate on same location in images. I wonder what mistakes may  cause this? Thank you!

**Comment by crockwell** [comment]
Can you please clarify? Are you saying that keypoints predicted are at the same location in all test images? How is your training and validation loss?

**Comment by acmff22** [comment]
I trained network using synthetic images like these, the objects on images are rendered using 3D models.
![synthetic](https://user-images.githubusercontent.com/37356249/82075978-b11d1c00-970f-11ea-98e7-22714671ba61.jpg)

the losses are low and I test the model using images blow, the objects in these images are also rendered, the results are good
![near](https://user-images.githubusercontent.com/37356249/82075482-e4ab7680-970e-11ea-8cb8-7702708545b6.jpg)

but when I test the model using real images, like these.
![real](https://user-images.githubusercontent.com/37356249/82076008-bd08de00-970f-11ea-91b9-02d91e28cf03.jpg)
For all test images, the predicted keypoints are located on the center of images. I know that difference between real and synthetic images will cause the bad results, but why the predicted keypoints on real images will locate on the center of images?

**Comment by crockwell** [comment]
Sorry, on the top of my head I do not know exactly what could be causing your error. I'd recommend visualizing heatmap predictions in both cases to see what is going wrong. Perhaps validating with real images during training would give you some ideas on what is happening.
